maraboupy/test/test_tf.py

```diff
@@ -80,7 +89,7 @@ def test_conv_NCHW():
     network_nchw = Marabou.read_tf(filename)
 
     # Evaluate test points using both Marabou and Tensorflow, and assert that the max error is less than TOL
-    testInputs = [[np.random.random(inVars.shape) for inVars in network_nhwc.inputVars] for _ in range(NUM_RAND)]
+    testInputs = [[np.random.random(inVars.shape) for inVars in network_nhwc.inputVars] for _ in range(5)]
```

Comment:
Maybe reduce the value of NUM_RAND instead?

Reply:
I lowered NUM_RAND  to 5